File: arrays/array_input_elements/array_input_elements.java
/**
 * this code takes the elements of array as input
 * using different methods in Java language
 * 
 */

package array_input_elements;
import java.util.*;
public class array_input_elements{
    public static void arr_inp(int a[]){
        // initialise the scanner object
        Scanner sc = new Scanner(System.in);
        //take the elements of array as input
        for (int i=0;i<a.length;i++){
            System.out.println("Enter the element at index "+i+":");
            a[i] = sc.nextInt();
            
        }

        //display the new array
        System.out.println("Array after entering the elements:");
        for(int i=0;i<a.length;i++){
            System.out.println("The element at index "+i+" is "+a[i]);
        }

        sc.close();

            
    }

    public static void main(String args[]){
        int[] a = new int[5];
        System.out.println("Array before entering the elements:");
        for(int i=0;i<a.length;i++){
            System.out.println("The element at index "+i+" is "+a[i]);
        }
        arr_inp(a);
        
    }
}

File: arrays/array_input_elements/array_input_elements.c
/**
 * this code takes the elements of array as input
 * using different methods in C language
 * 
 */

#include <stdio.h>
void arr_inp(int *a,int n){
    
    //take elements of array as input
    printf("Enter the elements of array: \n");
    for (int i=0;i<n;i++){
        printf("Enter the element at index %d: ",i);
        scanf("%d",&a[i]);
    }

    //display the array
 
    for (int i=0;i<n;i++){
        printf("The element at index %d is %d\n",i,a[i]);
        
    }



}

int main(){
    int a[5];
    int n = sizeof(a)/sizeof(int);
    arr_inp(a,n);
    return 0;
}
